vmdb/app/controllers/api_controller.rb

```diff
@@ -95,6 +96,10 @@ def initialize
     @api_token_mgr   = TokenManager.new(@module)
   end
 
+  def self.all_encrypted_attributes
+    (ATTR_TYPES[:encrypted] | ::Vmdb::ConfigurationEncoder::PASSWORD_FIELDS).sort
```

Comment:
these are embedded in the yaml returning from the api.
so while it is not coming back as a column, it is a key within settings in those containers.
And those yaml hashes are coming over the wire in things like `provisioning_requests`